.github/workflows/publish.yml

```diff
@@ -38,6 +38,6 @@ jobs:
           sed -E -i '' $'s/(# Unreleased)/\\1\\\n\\\n# Release '"$VERSION/" RELEASES.md
           git commit -am "Release $VERSION"
           git tag "v$VERSION"
-          git push --tags
           cargo workspaces publish --from-git
+          git ps --tags
```

Comment:
Is this supposed to be `git push --tags`?

Reply:
@Mcat12 Good catch. https://github.com/rust-lang/chalk/pull/484